.github/workflows/issue-management.yml

```diff
@@ -0,0 +1,21 @@
+name: Issue management
+
+on:
+  schedule:
+  - cron: '41 8 * * *'
```

Comment:
Curious about the thoughts about this time, I can get behind 8:41am pst but these crons observe utc, so we should adjust this if that was the intent (Unless you really want 1am pst?)

Reply:
ah, Actions started doing that in template workflows because they were DDoSing themselves by setting every workflow to run at, 9am Pacific I think 😂. Lemme set to something early morning Pacific.